Code_Exercises/Exercise_18_Local_Memory_Tiling/solution.cpp

```diff
@@ -23,18 +23,15 @@
 #include <benchmark.h>
 #include <image_conv.h>
 
-
 class image_convolution;
 
 inline constexpr util::filter_type filterType = util::filter_type::blur;
 inline constexpr int filterWidth = 11;
 inline constexpr int halo = filterWidth / 2;
 
 TEST_CASE("image_convolution_tiled", "local_memory_tiling_solution") {
-  const char* inputImageFile =
-      "../Images/dogs.png";
-  const char* outputImageFile =
-      "../Images/blurred_dogs.png";
+  const char *inputImageFile = "../Images/dogs.png";
+  const char *outputImageFile = "../Images/blurred_dogs.png";
```

Comment:
While you are here, this could be `constexpr auto`.

Reply:
Change made.